package-testing/dotnet-sdk-relay/EppoSDKRelay/DTO/AttributeSet.cs

```diff
@@ -23,6 +17,5 @@ public class AttributeSet
     public Dictionary<string, object?> NumericAttributesAsNumbers => NumericAttributes
                 .Where(kvp =>
                     (kvp.Value is JsonElement jsonElement) && jsonElement.ValueKind == JsonValueKind.Number)
-                .ToDictionary(kvp => kvp.Key, static kvp => (object?)((JsonElement)kvp.Value).GetDouble());
-
+                .ToDictionary(kvp => kvp.Key, static kvp => kvp.Value == null ? null : (object?)((JsonElement)kvp.Value).GetDouble());
```

Comment:
JsonParsing to Object is awkward in dotnet, in any JSON library.
Here, we have a map of objects where each object is a JsonElement that wraps values, even of primitive types by adding a "ValueKind" property. This throws off the numeric checks in the Eppo SDK which relies on strict primitives being passed.
So, here, we cast to the JsonElement, then to Object to get the "primitive" then to a proper double.